### PR TITLE
Add database_id param to delete file

### PR DESCRIPTION
**Pull request by yusukefs**
## What?

api-file-provider の /delete に database_id パラメータを追加
POST /download の database_id パラメータに required を付与

## Why?

API実装の変更に追従

## See also [Optional]

Issue: https://github.com/dataware-tools/dataware-tools/issues/72
API実装変更: https://github.com/dataware-tools/api-file-provider/pull/26

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/protocols/pull/49#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fprotocols).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/protocols/blob/master/OWNERS)~~ [yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by d-hayashi — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/protocols/pull/49#pullrequestreview-746715606" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/protocols/pull/49#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fprotocols).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/protocols/blob/master/OWNERS)~~ [d-hayashi,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
